### ISSUE TITLE
Update references to other Lupin stories?

### ISSUE THREAD

**Issue by tuskpot** [standardebooks/maurice-leblanc_the-hollow-needle_alexander-teixeira-de-mattos#2]
There are several [endnotes](https://standardebooks.org/ebooks/maurice-leblanc/the-hollow-needle/alexander-teixeira-de-mattos/text/endnotes) that point the reader to previous Lupin stories. Most of these are available from Standard Ebooks, but under different titles.

- Some stories are attributed to _The Seven of Hearts_. In Standard Ebooks' library, this collection is titled [The Extraordinary Adventures of Arsène Lupin, Gentleman Burglar](https://standardebooks.org/ebooks/maurice-leblanc/the-extraordinary-adventures-of-arsene-lupin-gentleman-burglar/george-morehead).
- One story is attributed to _The Exploits of Arsène Lupin_, which appears to be the same collection as above. (I don't know why they referenced it under two different names.)
- One of the stories is referenced as _Holmlock Shears Arrives Too Late_. In Standard Ebooks' edition, that story uses the title as it was originally published, _Sherlock Holmes Arrives Too Late_, from before Leblanc had to change it.
- A couple endnotes reference _Arsène Lupin Versus Holmlock Shears_. In Standard Ebooks' library, that novel is titled [Arsène Lupin Versus Herlock Sholmes](https://standardebooks.org/ebooks/maurice-leblanc/arsene-lupin-versus-herlock-sholmes/george-morehead), using a different copyright-evading name for the detective than the one that is used in _Hollow Needle_.

So, I guess I have two questions:

1. Would you accept a PR to update these references to match the Standard Ebooks titles?
2. Or, perhaps it would be fine to just cut the endnotes that reference prior stories? They don't really seem all that important.
3. Separately, is it worth making the names for Holmes/Sholmes/Shears consistent across the Lupin corpus? Especially since the characters refer back to previous stories, it seems like it would be better if they had consistent names. Since Holmes is now well out of copyright, it seems like we could just update everything to Sherlock Holmes, as Leblanc originally intended. Or else update everything to Holmlock Shears, which is the name that was eventually settled on? I'm happy to work on that change (in either direction) if it would be of interest.

**Comment by acabal** [comment]
@robinwhittleton thoughts?

I think an argument could be made to make the Holmes references consistent, but don't think we should change it to "Sherlock Holmes". That's not how the stories are known and it's somehow much less delightful.

**Comment by robinwhittleton** [comment]
More of the SE books were translated by Teixeira de Mattos, with the exception of the first two. I’d be happy for the endnotes to match the titles. Not sure I’d want to cut any: I think references to previous stories add enough value to remain.

As for standardising the name, if we do decide to do it the easiest would be to go to “Herlock Sholmes”, as if we changed it to anything else we’d need to set up URL redirects, etc. I have a bit of a soft spot for Herlock anyway 🙂 But honestly, I’m not sure that that change is worth it. I can’t see there being any reader that is interested enough in the interconnected nature of the books that also doesn’t realise that Herlock Sholmes / Holmlock Sheares is a exceedingly-lightly veiled reference to Sherlock Holmes.

**Comment by acabal** [comment]
OK, so let's update the title refs and standardize to Herlock Sholmes. The Wikipedia article says it was changed to Holmlock Shears for the US publication only. I think we have all of the Herlock Sholmes stories and there aren't any more in existence, correct?

 @tuskpot can you put together PRs?

**Comment by tuskpot** [comment]
Sounds good. I will start putting together PRs over the next couple weeks.

**Comment by tuskpot** [comment]
Okay, I think I've found and updated all the appropriate references to Holmes/Sholmes/Shears. I checked the endnotes for all the books and updated references to match the Standard Ebooks titles, as well as linking to the books that are available. I mentioned this issue in each PR, so it looks like Github has helpfully linked them all in the conversation above.

Note that Leblanc occasionally references "Sherlock Holmes as fictional detective" as a distinct entity from "Lupin rival Herlock Sholmes". This looks like Leblanc just playing around, or maybe teasing/pacifying Doyle. I left these as-is and noted them in the pull requests as well as the production notes.

I'll also note that I'm on a new workstation and haven't been able to install the Standard Ebooks tools due to this [toolset issue](https://github.com/standardebooks/tools/issues/561), so I haven't run `se lint` or anything to make sure I didn't break anything. If you want to wait until the next toolset release, I can do that review you merge these in. (I don't really expect any problems, but I failed to close a tag correctly in a recent PR, which caused a build error that someone else had to report, and I would rather not make a habit of that.)
